.github/actions/kfp-cluster-k8s-native/action.yml

```diff
@@ -0,0 +1,26 @@
+name: "Set up KFP on KinD"
```

Comment:
Can you just add an input to `.github/actions/kfp-cluster/action.yml` for optionally setting `--deploy-k8s-native` rather than duplication the action?

Reply:
I added an option to the script to alternatively deploy KFP w/ K8s Native API